subprojects/plugins/src/main/java/org/gradle/api/plugins/JavaTestFixturesPlugin.java

```diff
@@ -60,31 +59,22 @@ public void apply(Project project) {
                     .exposesApi()
                     .published()
             );
-            createImplicitTestFixturesDependencies(project, findJavaExtension(project));
+            createImplicitTestFixturesDependencies(project);
         });
     }
 
-    private void createImplicitTestFixturesDependencies(Project project, JavaPluginExtension extension) {
+    private void createImplicitTestFixturesDependencies(Project project) {
         DependencyHandler dependencies = project.getDependencies();
         dependencies.add(TEST_FIXTURES_API, dependencies.create(project));
-        SourceSet testSourceSet = findTestSourceSet(extension);
+        SourceSet testSourceSet = JvmPluginsHelper.getDefaultTestSuite(project).getSources();
```

Comment:
❔ Applying the text fixtures plugin doesn't seem to automatically apply any other plugins.  This is odd, shouldn't it apply java-base as least, and jvm-test-suites if it is going to rely on the default test suite existing.

Seems kind of reasonable to me that you could have a test fixtures project without tests or without any Java functionality.  Maybe your fixture is a bunch of resource files?

Reply:
This code is wrapped in a callback which is only run when the java plugin is applied. IMO future iterations of test fixtures will be merged with the TestingBasePlugin, allowing us to have a `fixtures` block within the `testing` block, alongside `suites`.